adapters/oidc/adapter-core/pom.xml

```diff
@@ -143,7 +148,8 @@
                           osgi.extender; filter:="(osgi.extender=osgi.serviceloader.processor)", osgi.serviceloader; filter:="(osgi.serviceloader=org.keycloak.adapters.authorization.ClaimInformationPointProviderFactory)";cardinality:=multiple, osgi.extender; filter:="(osgi.extender=osgi.serviceloader.registrar)"
                         </Require-Capability>
                         <Provide-Capability>
-                          osgi.serviceloader; osgi.serviceloader=org.keycloak.adapters.authorization.ClaimInformationPointProviderFactory
+                            osgi.serviceloader;
+                            osgi.serviceloader=org.keycloak.adapters.authorization.cip.spi.ClaimInformationPointProviderFactory
```

Comment:
Maybe it is a detail, but are you sure that it works when you use 2 separate lines? Wonder if it is still a bit safer to use the 1 line as before - just in case? So something like:
```
osgi.serviceloader; osgi.serviceloader=org.keycloak.adapters.authorization.cip.spi.ClaimInformationPointProviderFactory
```
But if you're sure that it works, I am fine with keep as you have it here.

Reply:
Reverted